Arm/rover_arm_system.hpp

```diff
@@ -17,22 +17,22 @@ class RoverArmSystem : public sjsu::common::RoverSystem
       kDefault = 'D',
     };
     Modes modes = Modes::kDefault;
-    int rotunda_speed;
-    int rotunda_angle;
-    int shoulder_speed;
-    int shoulder_angle;
-    int elbow_speed;
-    int elbow_angle;
-    int wrist_speed;
-    int wrist_roll;
-    int wrist_pitch;
+    double rotunda_speed;
+    double rotunda_angle;
+    double shoulder_speed;
+    double shoulder_angle;
+    double elbow_speed;
+    double elbow_angle;
+    double wrist_speed;
+    double wrist_roll;
+    double wrist_pitch;
     struct Finger
     {
-      int pinky;
-      int ring;
-      int middle;
-      int pointer;
-      int thumb;
+      double pinky;
+      double ring;
+      double middle;
+      double pointer;
+      double thumb;
```

Comment:
This might be problematic since we've had troubles in the past parsing doubles from the GET request response but it is fine for now. Maybe when @tbejaoui1 or @acarrou start work on arm --> mission control logic we will find out and can change later

Reply:
Got it, I just wanted to stay consistent with the types I was using, I'll change those in the future when needed.